content/platforms/kubernetes/db-controller.md

```diff
@@ -0,0 +1,274 @@
+---
+Title: Managing databases in Kubernetes
+description: The database controller provides the ability to create, manage,
+  and use databases via a database custom resource.
+weight: 35
+alwaysopen: false
+categories: ["Platforms"]
+aliases: /rs/concepts/kubernetes/k8s-operator-based-deployments
+---
+A database is created with a custom resource just like a cluster.
+The database controller in the Redis Enterprise operator:
+
+- Discovers the custom resource
+- Makes sure that it is created on the referenced cluster
+- Maintains parity between its specification and the database within the Redis Enterprise cluster
+
+The custom resource defines the size and other facets of the desired database.
+For example, a 1GB database can simply be created on the `rec` cluster with the resource:
+
+```yaml
+kind: RedisEnterpriseDatabase
+metadata:
+  name: smalldb
+spec:
+  memory: 1GB
+  redisEnterpriseCluster:
+    name: rec
+```
+
+The cluster is referenced by name in the `redisEnterpriseCluster` and must exist in the same namespace.
+Then, the database is created when you apply the resource file (for example, `smalldb.yaml`):
+
+```sh
+kubectl apply -f smalldb.yaml
+```
+
+The database controller recognizes the new custom resource and validates the specification.
+If the database specification is valid, the controller creates the database on the specified Redis Enterprise cluster.
+The database is exposed with the same service mechanisms by the service rigger for the Redis Enterprise cluster.
+
```

Comment:
Here, or later when we provide spec reference, we need to state clearly that spec options which are omitted use default values (which you refer to in the reference). Might seem obvious to some but not for all audiences. This is also a powerful message, as it is easy to get started with a minimal spec.

Reply:
I updated the introduction to address this. Please review.